src/components/capture/create/index.tsx

```diff
@@ -52,47 +53,73 @@ const selectors = {
         state.spec.errors,
     ],
     specFormData: (state: CaptureCreationState) => state.spec.data,
-    disoverLink: (state: CaptureCreationState) =>
-        state.links.discovered_catalog,
-    hasConnectors: (state: CaptureCreationState) => state.hasConnectors,
+    connectors: (state: CaptureCreationState) => state.connectors,
 };
 
+interface ConnectorTag {
```

Comment:
There is a `ConnectorTag` interface defined in _src/components/capture/create/Spec.tsx_ as well. Looks like you are leaning toward using generic names for interfaces that are local to files. Is that an accurate conclusion?